web-client/src/presenter/actions/setUserAction.test.js

```diff
@@ -32,14 +32,5 @@ describe('setUserAction', () => {
     expect(applicationContext.setCurrentUser.mock.calls[0][0]).toMatchObject(
       user,
     );
-    expect(
-      applicationContext.getUseCases().setItemInteractor.mock.calls.length,
-    ).toEqual(1);
-    expect(
-      applicationContext.getUseCases().setItemInteractor.mock.calls[0][1],
-    ).toMatchObject({
-      key: 'user',
-      value: user,
-    });
```

Comment:
The name of this test is `stores the user onto local storage when running local` with the operative part being `running local`.  So, should we either rename this test to not state we are running locally or should we set the `process.env.IS_LOCAL` to `true` (and clear it afterward) and re-instate this deleted code?